src/main/java/com/reverse/userservice/models/ProfilePicture.java

```diff
@@ -0,0 +1,41 @@
+package com.reverse.userservice.models;
+
+import com.sun.deploy.ui.ImageLoader;
```

Comment:
Please remove this import, it is causing problems in Jenkins. Thanks!
(com.sun.deploy.ui.ImageLoader)